watsong/structures.py

```diff
@@ -3,10 +3,14 @@
 It also includes some generic types that can be used for error handling, such as Result.
 """
 
-
 from typing import NamedTuple, List, Tuple, TypeVar, Optional
 from typing_extensions import TypedDict  # because we support 3.6.5
 
+Feel = NamedTuple(
+    "Feel",
+    [("energy", float), ("lyrics", float), ("dance", float), ("melody", float)],
+)
+
```

Comment:
I added this while creating the filter API. Might cause merge conflicts so maybe best to leave it out for now?

Reply:
Sounds good, I removed it since it isn't really in the scope of this PR. Though in general I'm not that afraid of merge conflicts, it's just as hard to go back and remove something as to fix a merge